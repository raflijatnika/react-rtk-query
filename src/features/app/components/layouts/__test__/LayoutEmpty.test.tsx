// Test
import { render, screen } from '@testing-library/react'

// Components
import { AppLayoutEmpty } from '../AppAppLayoutEmpty'

it('AppLayoutEmpty should be rendered', () => {
  const { container } = render(
    <AppLayoutEmpty>
      <h1 data-testid='content'>Content</h1>
    </AppLayoutEmpty>
  )
  const element = screen.getByTestId('content')
  expect(container.innerHTML).toMatchSnapshot()
  expect(element.textContent).toBe('Content')
})
