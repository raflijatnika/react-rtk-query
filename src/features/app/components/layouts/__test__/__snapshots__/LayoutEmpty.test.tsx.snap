// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutEmpty should be rendered 1`] = `"<div class=\\"sc-bczRLJ hUFkAN\\"><h1 data-testid=\\"content\\">Content</h1></div>"`;
